lib/internal/source_map/source_map_cache.js

```diff
@@ -148,17 +150,39 @@ function rekeySourceMap(cjsModuleInstance, newInstance) {
   }
 }
 
+// WARNING: The `sourceMapCacheToObject` and `appendCJSCache` run during
+// shutdown. In particular, they also run when Workers are terminated, making
+// it important that they do not call out to any user-provided code, including
+// built-in prototypes that might have been tampered with.
+const ObjectCreate = Object.create;
+const ObjectKeys = Object.keys;
+const ObjectGetOwnPropertyDescriptor = Object.getOwnPropertyDescriptor;
+const ObjectHasOwnProperty = uncurryThis(Object.prototype.hasOwnProperty);
```

Comment:
we already have this in `primordials.ObjectPrototype.hasOwnProperty`

Reply:
@targos Thanks, switched to using that